src/server/app.js

```diff
@@ -0,0 +1,70 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var requestLogger = require('./lib/requestLogger');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var compression = require('compression');
+var config = require('./config');
+
+var routes = require('./routes/index');
+var proxy = require('./routes/proxy');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
+
+// uncomment after placing your favicon in /public
+//app.use(favicon(__dirname + '/public/favicon.ico'));
```

Comment:
We just need to uncomment that line and add the path to the favicon. I'm not sure this is really required since we link directly to it in the HTML.